displayimages.css

```diff
@@ -1,4 +1,34 @@
 .table_image{
 	width : auto;
 	height : 8em;
+}
```

Comment:
Just an FYI, I'm not able to review CSS or javascript, but I'm happy to trust you.